python-sdk/dev/docker-compose.yaml

```diff
@@ -21,6 +21,8 @@ x-airflow-common:
     ASTRONOMER_ENVIRONMENT: local
     AIRFLOW__LINEAGE__BACKEND: openlineage.lineage_backend.OpenLineageBackend
     OPENLINEAGE_URL: http://host.docker.internal:5050/
+    OPENLINEAGE_NAMESPACE: "astro_dev"
```

Comment:
Why do we need this?

Reply:
I will change this to `astro`. It points to open lineage default namespace otherwise

Reply:
[Done](https://github.com/astronomer/astro-sdk/pull/1025/commits/7dad9a6fc4f0bcc5205f25fe51fb0b3f629a621e)

Reply:
Does a user need to set this? or is it set for them?

Reply:
By default, it is set to open lineage default namespace. User can change that to their own namespace of their choice with`OPENLINEAGE_NAMESPACE`.